node_lambnik/src/tiler/src/tile-interface.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Handles accepting requests from the api
+ * and sending responses using mapnik
+ *
+ * TODO: this module could use a better name?
+ */
+
+import path from 'path'
+import fs from 'fs'
+
+import { image, grid } from './tiler'
+
+
+// TODO: flesh out how errors are handled
```

Comment:
On Urban Apps, we recently had some discussions about TODO items and arrived at a process whereby any TODO added to the code (i.e. anything you're planning to merge, not just a note to be handled before finalizing a PR) should have a corresponding issue in the repo, with the issue number included in the TODO comment.  I think it's a good system, and that these TODO items are worth adding issues for.

Reply:
Sounds good, I'll add those in.